spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubAcknowledger.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.pubsub.core;
+
+import java.util.Collection;
+
+/**
+ * Acknowledges Pub/Sub messages based on their subscription name and ack ID.
+ *
+ * @author João André Martins
+ */
+public interface PubSubAcknowledger {
+
+	/**
+	 * Acknowledges a bulk of messages based on their ack IDs which pertain to a subscription name.
```

Comment:
The imperative style is lost: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting